src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -65,17 +65,19 @@ public partial class ArithmeticException : System.Exception
         public ArithmeticException(string message) { }
         public ArithmeticException(string message, System.Exception innerException) { }
     }
-    public abstract partial class Array : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList, System.Collections.IStructuralComparable, System.Collections.IStructuralEquatable
+    public abstract partial class Array : System.Collections.ICollection, System.Collections.IEnumerable, System.Collections.IList, System.Collections.IStructuralComparable, System.Collections.IStructuralEquatable, System.ICloneable
```

Comment:
Can you please verify that this represents all of the Array members now? Are there any more that aren't included here?